lib/redcord/server_scripts/shared/lua_helper_methods.erb.lua

```diff
@@ -30,3 +30,27 @@ local function set_list_intersect(set, list)
   end
   return set_intersect
 end
+
+local function adjust_string_length(value)
+  if value == '' or value == nil then
+    return '!'
+  end
+  local whole_digits_count = 10
+  local decimal_digits_count = 4
+  local sep = '.'
+  local parts = {}
+  for part in string.gmatch(value, "([^"..sep.."]+)") do
```

Comment:
hmm what if people have a string attr and it happens to be in the custom index, also it has a `sep` in it?

Reply:
would it be easier if we calculate this custom index string in ruby (and make the format (digit count ) configurable by each model?)

Reply:
nvm we don't support string!

Reply:
added validation for that.